app/src/main/java/org/p2p/wallet/home/ui/main/HomeFragment.kt

```diff
@@ -169,10 +188,14 @@ class HomeFragment :
 
     override fun showEmptyState(isEmpty: Boolean) {
         with(binding) {
-            emptyStateLayout.isVisible = isEmpty
-            swipeRefreshLayout.isVisible = !isEmpty
+            actionButtonsView.isVisible = !isEmpty
             balanceTextView.isVisible = !isEmpty
             balanceLabelTextView.isVisible = !isEmpty
+            mainRecyclerView.adapter = if (isEmpty) {
+                emptyAdapter
+            } else {
+                mainAdapter
```

Comment:
we can rename `mainAdapter` to `contentAdapter` so it the screen state would be more explicit